bip-satoshi-standard.mediawiki

```diff
@@ -0,0 +1,88 @@
+<pre>
+BIP: TBD
+Layer: Applications
+Title: Define Bitcoin Subunits as Satoshis
+Author: OceanSlim <oceanslim@gmx.com>
+Comments-Summary: No comments yet.
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-TBD
+Status: Draft
+Type: Informational
+Created: 2025-04-28
+License: BSD-2-Clause
+</pre>
+
+== Abstract ==
+This BIP proposes to formally define and standardize Bitcoin's smallest indivisible unit (1/100,000,000 of a bitcoin) as "satoshi" (singular) or "satoshis" (plural), with "sats" as the standard abbreviated form. This standardization aims to improve clarity in communication, user interfaces, documentation, and development across the Bitcoin ecosystem.
+
+
+== Motivation ==
+As Bitcoin adoption grows, the need for a clear, standardized terminology for its subunits becomes increasingly important. Currently, there are various terms used to refer to Bitcoin's smallest unit (1/100,000,000 BTC), including "satoshi," "satoshis," "sat," "sats," and sometimes simply expressed as decimal values of bitcoin (0.00000001 BTC).
+
+This lack of standardization can lead to:
+
+1. Confusion for newcomers to the Bitcoin ecosystem
+2. Inconsistent user experiences across different applications and services
+3. Ambiguity in technical documentation and discussions
+4. Potential errors in development and implementation of Bitcoin-related software
+
+By formally establishing "satoshi"/"satoshis" as the standard term with "sats" as the recognized abbreviation, we can improve clarity and consistency throughout the ecosystem.
+
+== Specification ==
+This BIP formally defines:
+
+1. The term "satoshi" (singular) or "satoshis" (plural) shall be the standard term for 1/100,000,000 of a bitcoin (0.00000001 BTC).
+
+2. The abbreviation "sat" (singular) or "sats" (plural) shall be the standard short form for "satoshi"/"satoshis".
+
+3. The relationship between units shall be expressed as:
+   - 1 bitcoin (BTC) = 100,000,000 satoshis
+   - 1 satoshi = 0.00000001 bitcoin
+
+4. The terms "satoshi" and "sats" are not proper nouns when referring to the unit of currency and should not be capitalized except at the beginning of sentences or in titles.
+
+5. Bitcoin-related applications, services, documentation, and communications are encouraged to use these standardized terms to promote consistency across the ecosystem.
+
+== Rationale ==
```

Comment:
The rationale could be extended to address BIP 176 and BIP 177 as alternate approaches.